### ISSUE TITLE
Add extra roles for components

### ISSUE THREAD

**Issue by MagnusRef** [projectsveltos/helm-charts#42]
In my use case Im gonna be using Sveltos with crossplane to provision Terraform.

The problem is that the standard clusterRoles for the addon-controller does not cover the CRDs from the Terraform provider for crossplane.

It would be cool to be able to add more items to the clusterRoles

**Comment by gianlucam76** [comment]
@MagnusRef when Sveltos needs extra permissions in the management cluster (the case you described above), this "controller-role-extra" ClusterRole can be used.

Sveltos addon-controller is tied to it. Currently ClusterRole _controller-role-extra_ grants no extra permissions. But if you can modify that manually for now.

Do you want those permissions to be a configuration values for helm?

**Comment by MagnusRef** [comment]
> @MagnusRef when Sveltos needs extra permissions in the management cluster (the case you described above), this "controller-role-extra" ClusterRole can be used.
> 
> Sveltos addon-controller is tied to it. Currently ClusterRole _controller-role-extra_ grants no extra permissions. But if you can modify that manually for now.
> 
> Do you want those permissions to be a configuration values for helm?

Yes I used to create my own role for this, however it would be neat to have this as part for the helm chart templating.

But with the introduction of kusomization as a installation option soon, it isn't something I would be missing.

Can be closed or backlogged.

**Comment by jkroepke** [comment]
Seems to be fixed in #143
